addons/sales_team/static/src/scss/crm_team_views.scss

```diff
@@ -13,9 +13,11 @@ $o-kanban-medium-record-width: 350px;
 .o_kanban_view.o_crm_team_member_kanban {
     .o_kanban_group:not(.o_column_folded) {
         width: $o-kanban-medium-record-width + 2*$o-kanban-group-padding;
+        min-height: 100px;
```

Comment:
Should this fix be done in stable or is it a small UI glitch we don't want to backport ?

Reply:
It is really a small glitch, without it "archived" banned is a bit greater than the card itself. Don't think it is worth a fix in stable.